src/angular/header-lean/app-chooser-section.scss

```
@@ -47,4 +47,25 @@
   &:last-child {
     border-bottom: var(--sbb-border-width-thin) solid var(--sbb-color-cloud);
   }
+
+  .sbb-icon,
+  img {
+    vertical-align: middle;
+    width: var(--sbb-icon-size-default);
+    height: var(--sbb-icon-size-default);
+
+    // adjust margin to keep line height
```

```suggestion
    // Adjust margin to keep line height
```
We once started to write comment starting letters upper case